tests/performance/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+cmake_minimum_required(VERSION 3.5)
+
+set(OLP_SDK_PERFORMANCE_TESTS_SOURCES
+    ./PerformanceTest.cpp
+)
+
+if (ANDROID OR IOS)
```

Comment:
Why do we need android and ios? remove.

Reply:
Why not? ios and android devices are real hardware and memory/cpu data very representative.

Reply:
As we do not execute them yet on CI. If we will integrate iOS & Android tests in FV/NV on CI we can add the additional cmake entries pretty quick.

Reply:
We don't execute anything on iOS/Android PSV/NV/etc. But our cmake offers you to generate test app for that. At least now we can do that manually.

Reply:
At least to me as a user of the SDK tests are not very interesting.
Did you ever execute the tests from boost, leveldb, rapidjson or any other public repo that you used?

Reply:
Yes, we should. But not part of this ticket.